libcudacxx/codegen/CMakeLists.txt

```
@@ -1,44 +1,44 @@
 ## Codegen adds the following build targets
 #    libcudacxx.atomics.codegen
-#    libcudacxx.atomics.codegen.execute
 #    libcudacxx.atomics.codegen.install
 ## Test targets:
-#    libcudacxx.atomics.codegen.diff
+#    libcudacxx.test.atomics.codegen.diff
 
-add_custom_target(libcudacxx.atomics.codegen)
+include(${CMAKE_SOURCE_DIR}/cub/cmake/CPM.cmake)
+CPMAddPackage("gh:fmtlib/fmt#10.2.1")
 
 add_executable(
     codegen
     EXCLUDE_FROM_ALL
     codegen.cpp
 )
 
-target_compile_features(
-    codegen PRIVATE cxx_std_14
-)
+target_link_libraries(codegen fmt)
```

suggestion:

PUBLIC/PRIVATE/INTERFACE linkage should be explicit when linking libraries.

```suggestion
target_link_libraries(codegen PRIVATE fmt)
```